### ISSUE TITLE
Serpentine skin lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9949]
**Preferred term label:**
Serpentine skin lesions

**Synonyms**
snake-like shape

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Granuloma inguinale

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000530
- label: Serpentine skin location
- parents: [Position (HP:0012830)](https://hpo.jax.org/app/browse/term/HP:0012830)
- pmid: PMID:30020678
- def: Refers to a skin lseion that has a snake-like (serpentine) shape.
- comment: Lesions of granuloma inguinale are said to have a serpentine appearance.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9949
- synonyms:
